shelter thingy/resources/css/md.css

```diff
@@ -1,449 +1,461 @@
+	height: 40px;
+	width: 40px;
 }
 
-input{
-    background-color: white;
+input {
+	background-color: white;
 }
+
 input:focus {
-    border: 5px solid #84BD00;
-}
-
-@media screen and (min-width: 0px) and (max-width:992px){
-    .card{
-        height: auto;
-    }
-}
-
-@media screen and (min-width:1000px) {
-    .hero-content{
-        font-size: 4rem;
-    }
-}
-
-@media screen and (max-width:639px){
-    .top {
-        position: absolute;
-        top: 0;
-    }
-
-
-    .bottom {
-        bottom:0;
-    }
-
-    .parent {
-        height: 468px;
-    }
-
-    .hero-content{
-        position: absolute;
-        left: 67px;
-    }
-
-    .blackBox {
-        position: absolute;
-        left: 0;
-        right:0;
-    }
-    
-    .right {
-        position: absolute;
-        right: 0;
-        left:0;
-        
-    }
-    
-}
-@media screen and (max-width: 398px) and (min-width: 200px){
-    .hero-text {
-        transform: translate(-50%, -140%);
-    }
-    #differenceform {
-        transform:translate(-45%, 80%);
-    }
-    .form-control-lg {
-        width: 90%;
-        margin-top: 10px;
-    }
-
-    .hero-image{
-        height: 1000px;
-    }
-
-}
-
-@media screen and (max-width: 590px) and (min-width: 398px){
-    .hero-text {
-        transform: translate(-50%, -120%);
-    }
-    #differenecform{
-        transform:translate(-45%, 70%);
-    }
-    .form-control-lg {
-        width: 90%;
-        margin-top: 10px;
-    }
-
-    .hero-image{
-        height: 800px;
-    }
-
-    .card {
-        height: auto;
-    }
-
-}
-
-@media screen and (max-width: 768px) and (min-width: 590px){
-    .hero-text {
-        transform: translate(-50%, -150%);
-    }
-    #differenceform {
-        transform:translate(-45%, 70%);
-    }
-    .form-control-lg {
-        width: 90%;
-        margin-top: 10px;
-    }
-
-    .hero-image{
-        height: 800px;
-    }
-
-}
-
-@media screen and (max-width:860px) and (min-width: 768px) {
-    
-    .hero-text {
-        transform: translate(-50%, -180%);
-    }
-
-    differenceform {
-        transform:translate(-45%, 140%);
-    }
-
-    .form-control-lg {
-        width: 300px;
-        margin-top: 10px;
-    }
-
-    #gridCheck1{
-        width: 80px;
-        height: 80px;
-    }
-    .form-inline .form-check-input{
-        margin-right: 2rem;
-    }
+	border: 5px solid #84bd00;
+}
+
+@media screen and (min-width: 0px) and (max-width: 992px) {
+	.card {
+		height: auto;
+	}
+}
+
+@media screen and (min-width: 1000px) {
+	.hero-content {
+		font-size: 4rem;
+	}
+}
+
+@media screen and (max-width: 639px) {
+	.top {
+		position: absolute;
+		top: 0;
+	}
+
+
+	.bottom {
+		bottom: 0;
+	}
+
+	.parent {
+		height: 468px;
+	}
+
+	.hero-content {
+		position: absolute;
+		left: 67px;
+	}
+
+	.blackBox {
+		position: absolute;
+		left: 0;
+		right: 0;
+	}
+
+	.right {
+		position: absolute;
+		right: 0;
+		left: 0;
+
+	}
+
+}
+
+@media screen and (max-width: 398px) and (min-width: 200px) {

```

Comment:
Ich finde es besser zu lesen, wenn die Argumente in der mobile-first-Reihenfolge stehen.
Also zuerst min-width, dann max-width

Das Betrifft auch einige andere Files